packages/common/src/tools/queue/TaskQueue.ts

```diff
@@ -35,6 +35,10 @@ export class TaskQueue<T> {
     return this.queue.length
   }
 
+  getBusyWorkers() {
+    return this.busyWorkers
+  }
```

Comment:
Why not set `readonly busyWorkers`?

Reply:
This will block updating this value from the inside of the class too.